src/tools/dev/scripts/visit-build

```diff
@@ -2,15 +2,20 @@
 set -e
 #-----------------------------------------------------------------------
 #
-# VISIT-BUILD-OPEN - Build the visit distributions on the open network.
+# VISIT-BUILD - Build the visit distributions on either the open or 
+#               closed network.
 #
 # Author: Eric Brugger
 # Date:   October 27, 2000
 #
 # Usage:
-#    visit-build-open -d <distribution>
+#    visit-build -d <distribution>
```

Comment:
do we really need to preserve *all* the modifications from both previous files?  Would just the mods from the past couple years be sufficient?
Not a PR buster, just offering food-for-thought.

Reply:
IMHO...ever since we moved to subversion, we should have abandon the practice of modification comments in the code. That is after all what a revision control system is supposed to do for us...capture the history of our work. But, we all rarely commit at the granularity where revision comments in the RCS would actually be useful.

Reply:
> do we really need to preserve all the modifications from both previous files? Would just the mods from the past couple years be sufficient?
> Not a PR buster, just offering food-for-thought.

I just didn't want to make anyone upset. I can delete them if you'd like.

> IMHO...ever since we moved to subversion, we should have abandon the practice of modification comments in the code. That is after all what a revision control system is supposed to do for us...capture the history of our work. But, we all rarely commit at the granularity where revision comments in the RCS would actually be useful.

I'd be in favor of ending revision comments and having a code sprint to delete all of them... would cut down on code lines for sure.

Reply:
I say we can punt on comment removal for now.